development/git_lfs.rst

```diff
@@ -0,0 +1,156 @@
+########################################################
+Using Git Large File Storage (LFS) for Data Repositories
+########################################################
+
+DM uses Git LFS to manage test datasets within our :doc:`normal Git workflow </development/workflow>`.
+`Git LFS is developed by GitHub <https://git-lfs.github.com/>`_, though DM uses its own backend storage infrastructure (see `SQR-001: The Git LFS Architecture <http://sqr-001.lsst.io>`_ for background).
+
+All DM repositories should use Git LFS to store binary data, such as FITS files, for :abbr:`CI (Continuous Integration)`.
+Examples of LFS-backed repositories in use are `lsst/afw <https://github.com/lsst/afw>`_, `lsst/hsc_ci <https://github.com/lsst/ci_hsc>`_ and `lsst/testdata_cfht <https://github.com/lsst/testdata_cfht>`_.
+
+This page describes how to use Git LFS for DM development.
+
+.. _git-lfs-install:
+
+Installing Git LFS
+==================
+
+Download and install the ``git-lfs`` client by visiting the `Git LFS <https://git-lfs.github.com>`_ homepage.
+If you downloaded the binary release, install ``git-lfs`` by running the provided :file:`install.sh`.
+
+Most package managers also provide the ``git-lfs`` client.
+Since, LFS is a rapidly evolving technology, package managers will help you keep up with new ``git-lfs`` releases.
+For example, Mac users with Homebrew_ can simply run ``brew install git-lfs`` and ``brew upgrade git-lfs``.
+
+Once ``git-lfs`` is installed, run
+
+.. code-block:: bash
+
+   git config --global lfs.batch false
+   git lfs install
+
+to install the necessary hooks in your :file:`~/.gitconfig` file.
+
+.. _git-lfs-auth:
+
+Setting up Git LFS authentication
+=================================
+
+Credentials *are not needed to clone or pull* from LFS-backed repositories, but are needed for push operations.
+Only GitHub users in the LSST GitHub organization can authenticate with DM's storage service.
+
+Git LFS mandates the HTTPS transport protocol.
+Since many HTTPS authentications happening during a single clone, push or fetch operation, it is essential that you use a Git credential helper to work effectively with Git LFS.
+:ref:`We describe how to setup a credential helper for you system in the deverloper workflow documentation <git-credential-helper>`.
+
+Once a helper is setup, you can cache your credentials by cloning any of DM's LFS-backed repositories.
+
+.. code-block:: bash
+
+   git clone https://github.com/lsst/afwdata.git
```

Comment:
You asked about a potentially smaller git-lfs repo. Maybe this will work:

https://github.com/lsst/testdata_cfht

or I hear a lot of discussion about decam:

https://github.com/lsst/testdata_decam